zktest/src/main/webapp/test2/B96-ZK-5253.zul

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zk>
+	<borderlayout >
```

Comment:
please update the test step (also in 9.6.3)

Reply:
Refine ZK-5253: add testcase description